CONTRIBUTING.md

```diff
@@ -4,20 +4,42 @@
     1. Merge an update to main with a new release number in the .json file. 
     2. Have someone from SecOps change the version in the Splunk SOAR interface.
 
+## Setup
+
+Python version should be set to 3.9 to match Splunk SOAR.
+
+```shell
+pyenv install 3.9.0
+pyenv local 3.9.0
```

Comment:
For consistency let's add dollar signs for the prompts:

```suggestion
$ pyenv install 3.9.0
$ pyenv local 3.9.0
```

Reply:
The Markdown style guides I have seen have indicate using a dollar sign when expected output is shown on the next line, and leaving it off when there is no output.